arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -1441,3 +1444,98 @@ def test_plot_bpv_discrete():
     fake_model = from_dict(posterior_predictive=fake_pp, observed_data=fake_obs)
     axes = plot_bpv(fake_model)
     assert not isinstance(axes, np.ndarray)
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {},
+        {
+            "binwidth": 0.5,
+            "stackratio": 2,
+            "nquantiles": 20,
+        },
+        {"point_interval": True},
+        {
+            "point_interval": True,
+            "dotsize": 1.2,
+            "point_estimate": "median",
+            "plot_kwargs": {"color": "grey"},
+        },
+        {
+            "point_interval": True,
+            "plot_kwargs": {"color": "grey"},
+            "nquantiles": 100,
+            "hdi_prob": 0.95,
+            "intervalcolor": "green",
+        },
+        {
+            "point_interval": True,
+            "plot_kwargs": {"color": "grey"},
+            "quartiles": False,
+            "linewidth": 2,
+        },
+    ],
+)
+def test_plot_dot(continuous_model, kwargs):
+    data = continuous_model["x"]
+    ax = plot_dot(data, **kwargs)
+    assert ax
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {"rotated": True},
+        {
+            "point_interval": True,
+            "rotated": True,
+            "dotcolor": "grey",
+            "binwidth": 0.5,
+        },
+        {
+            "rotated": True,
+            "point_interval": True,
+            "plot_kwargs": {"color": "grey"},
+            "nquantiles": 100,
+            "dotsize": 0.8,
+            "hdi_prob": 0.95,
+            "intervalcolor": "green",
+        },
+    ],
+)
+def test_plot_dot_rotated(continuous_model, kwargs):
+    data = continuous_model["x"]
+    ax = plot_dot(data, **kwargs)
+    assert ax
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {
+            "point_estimate": "mean",
+            "hdi_prob": 0.95,
+            "quartiles": False,
+            "linewidth": 2,
+            "markersize": 2,
+            "markercolor": "red",
+            "marker": "o",
+            "rotated": False,
+            "intervalcolor": "green",
+        },
+    ],
+)
+def test_plot_point_interval(continuous_model, kwargs):
+    _, ax = plt.subplots()
+    data = continuous_model["x"]
+    values = np.sort(data)
+    ax = plot_point_interval(ax, values, **kwargs)
+    assert ax
+
+
+def test_wilkinson_algorithm(continuous_model):
+    data = continuous_model["x"]
+    values = np.sort(data)
+    _, stack_counts = wilkinson_algorithm(values, 0.5)
+    assert np.sum(stack_counts) == len(values)
```

Comment:
Maybe a test designed after Figure 4 of Wilkinson's paper.
```suggestion
    assert np.sum(stack_counts) == len(values)
    stack_locs, stack_counts = wilkinson_algorithm(([0.0, 1.0, 1.8, 3.0, 5.0], 1.0)
    @assert stack_locs == np.array([0.0, 1.4, 3.0, 5.0])
    @assert stack_counts == np.array([1, 2, 1, 1])
```